hybridse/src/udf/default_udf_library.cc

```diff
@@ -310,6 +310,47 @@ struct DistinctCountDef {
     };
 };
 
+template <typename T>
+struct MedianDef {
```

Comment:
`median` needs end-to-end verify. 
May you try add a new case in `cases/function/function/test_udaf_function.yaml`, you can see the example like https://github.com/4paradigm/OpenMLDB/blob/af0272cec5b9646c8b7eaa3c6d9e9dfccaf6f56d/cases/function/function/test_udaf_function.yaml#L114-L143.  Docs is missing so don't afraid to ask.

Finally you can build the target `toydb_run_engine` and verify your newly case via:
```sh
./build/hybridse/examples/toydb/src/toydb_run_engine --yaml_path cases/function/function/test_udaf_function.yaml --case_id 4
```

I would be nice if you can add it

Reply:
> `median` needs end-to-end verify. May you try add a new case in `cases/function/function/test_udaf_function.yaml`, you can see the example like
> 
> https://github.com/4paradigm/OpenMLDB/blob/af0272cec5b9646c8b7eaa3c6d9e9dfccaf6f56d/cases/function/function/test_udaf_function.yaml#L114-L143
> 
> . Docs is missing so don't afraid to ask.
> Finally you can build the target `toydb_run_engine` and verify your newly case via:
> 
> ```shell
> ./build/hybridse/examples/toydb/src/toydb_run_engine --yaml_path cases/function/function/test_udaf_function.yaml --case_id 4
> ```
> 
> I would be nice if you can add it
ok, i try to add a end-to-end test case.